cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js

```diff
@@ -0,0 +1,63 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import { getTestEntityValue } from '../../../support/utils/stringTools';
+import Users from '../../../support/fragments/users/users';
+import PatronGroups from '../../../support/fragments/settings/users/patronGroups';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../../support/fragments/users/userEdit';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import SettingsFinance from '../../../support/fragments/settings/finance/settingsFinance';
+import ExpenseClasses from '../../../support/fragments/settings/finance/expenseClasses';
+import FundTypes from '../../../support/fragments/settings/finance/fundTypes';
+
+describe('Fund type view', () => {
+  let userData;
+  let servicePointId;
+  const patronGroup = {
+    name: getTestEntityValue('groupPermissions'),
+  };
+  const newFundTypes = { ...FundTypes.getDefaultFundTypes() };
+  const newExpenseClass = { ...ExpenseClasses.getDefaultExpenseClass() };
+
+  before('Preconditions', () => {
+    cy.getAdminToken().then(() => {
+      ServicePoints.getViaApi({ limit: 1, query: 'name=="Circ Desk 1"' }).then((servicePoints) => {
+        servicePointId = servicePoints[0].id;
+      });
+      PatronGroups.createViaApi(patronGroup.name).then((patronGroupResponse) => {
+        patronGroup.id = patronGroupResponse;
+      });
+      ExpenseClasses.createExpenseClassViaApi(newExpenseClass);
+      FundTypes.createFundTypesViaApi(newFundTypes);
+      cy.createTempUser([Permissions.uiSettingsFinanceView.gui], patronGroup.name).then(
+        (userProperties) => {
+          userData = userProperties;
+          UserEdit.addServicePointViaApi(servicePointId, userData.userId, servicePointId);
+          cy.login(userData.username, userData.password, {
+            path: SettingsMenu.fundTypesPath,
+            waiter: SettingsFinance.waitFundTypesLoading,
+          });
+        },
+      );
+    });
+  });
+
+  after('Deleting created entities', () => {
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    ExpenseClasses.deleteExpenseClassViaApi(newExpenseClass.id);
+    FundTypes.deleteFundTypesViaApi(newFundTypes.id);
+  });
+
+  it(
+    'C409416 A user with "Settings (Finance): View settings" permission can only view appropriate settings (Thunderjet)(TaaS)',
+    { tags: [TestTypes.criticalPath, DevTeams.thunderjet] },
+    () => {
+      SettingsFinance.verifyItemInFinancePanel();
+      SettingsFinance.verifyItemInDetailPanel();
+      SettingsFinance.checkFundType(newFundTypes);
+      SettingsFinance.clickExpenseClass();
+      SettingsFinance.verifyItemInDetailPanel();
+      SettingsFinance.checkExpenseClass(newExpenseClass);
```

Comment:
Last step in test case and last step here are differnt
"Expense classes" pane appears and contains list of created expense classes
"New" button on the top right is NOT active
"Actions" column is empty and does NOT contain "Edit" (pencil) and "Delete" (trash can) icons

Reply:
You need to add method, whitch will be check, tha user can not edit or delete expense classes

Reply:
verifyItemInDetailPanel() {
    cy.expect([
      actions.absent(),
      newButton.has({ disabled: true }),
    ]);
  },
  
  I have checked in this method, please confirm

Reply:
> You need to add method, whitch will be check, tha user can not edit or delete expense classes

beacuse I can not find the Action button,So we also can not find edit and delete icon in dropdown of action

Reply:
![Screenshot_343](https://github.com/folio-org/stripes-testing/assets/99126301/f623058a-8d47-4820-a8a3-7c75959de506)

Reply:
![Screenshot_344](https://github.com/folio-org/stripes-testing/assets/99126301/f46eb5ba-5aa6-4795-a2b5-fcec93abedb5)

Reply:
Add below function ,please check
checkEditDeleteIcon: (typeClass) => {
    cy.do(
      MultiColumnListCell({ content: typeClass.name }).perform((element) => {
        const rowNumber = element.parentElement.parentElement.getAttribute('data-row-index');
        cy.expect(getEditableListRow(rowNumber).find(editIconButton).absent());
        cy.expect(getEditableListRow(rowNumber).find(deleteButton).absent());
      }),
    );
  },